documentation/api/stacks.md

```diff
@@ -5,15 +5,17 @@
 
 Captures the call stacks of a target process. Note that only managed frames are collected.
 
->**Note**: This feature is not enabled by default and requires configuration to be enabled. The [in-process features](./../configuration/in-process-features-configuration.md) must be enabled since the call stacks feature uses shared libraries loaded into the target application for collecting the call stack information.
+> [!NOTE]
+> This feature is not enabled by default and requires configuration to be enabled. The [in-process features](./../configuration/in-process-features-configuration.md) must be enabled since the call stacks feature uses shared libraries loaded into the target application for collecting the call stack information.
```

Comment:
Outside of this PR:

I noticed that we still state the `stacks` and `exceptions` features are disabled by default throughout the docs. We should update these.